_data/processes.yml

```diff
@@ -0,0 +1,54 @@
+- title       : "Step 1: Meet and Greet"
+  image       : 1_meet_and_greet.png
+  description : "We listen to find out as much as we can about your business, target market,
+                 goals and budget. This is a business meeting, designed to figure out what you
+                 want and what you need. We’ll have some questions for you as well: we’ll need to
+                 know who’s going to use the application, how team members would use it – things
+                 like that – which will help us help you."
+- title       : "Step 2: Size and Scope"
+  image       : 2_size_and_scope.png
+  description : "You want to know what it’s going to cost and the timeline to get it done. We’ll
+                 move quickly to get you answers. We’ll huddle up internally and determine what we
+                 believe is the size and scope of a solution. We’ll talk with subject matter
+                 experts and our developers internally to come up with an initial quote."
+- title       : "Step 3: Present Your Options"
+  image       : 3_present_your_options.png
+  description : "We’ll show you a high-level breakdown of your application’s structure and offer
+                 options which might work well for you. You’ll likely have more questions, and we’ll
+                 address those in this meeting.  We’ll do any final sizing and scoping after that.
+                 The option you like best will result in a fully researched final quote and we’ll
+                 sign a contract."
+- title       : "Step 4: Wireframes"
+  image       : 4_wireframes.png
+  description : "A wireframe is a term in our business for the visual representation of the pages in
+                 your application: Where elements go, what they do, what they’ll do next. This shows
+                 you the basic structure of your application. This is the” look and feel” part of
```

Comment:
Adjust spacing:
the "look and feel"
vs
the" look and feel"

Reply:
Thanks for the catch. Fixed.